test/es6/destructuring_bugs.js

```diff
@@ -367,10 +367,6 @@ var tests = [
   {
     name: "Destructuring pattern at param has nested blocks.",
     body: function () {
-        assert.doesNotThrow(function () { eval("var e = 1;       ( {abcdef  = ((((({})) = (1))))} = (e)) => {  try{ } catch(e) {}}") }, "Should not throw when the default value and it's initializer both have extra parentheses.");

```

Comment:
These test are actually invalid syntax (you can't put parens around the "object" or "array" pattern).